src/Bridge/Doctrine/Orm/Extension/EagerLoadingExtension.php

```diff
@@ -40,10 +40,10 @@ public function __construct(PropertyNameCollectionFactoryInterface $propertyName
     public function applyToCollection(QueryBuilder $queryBuilder, QueryNameGeneratorInterface $queryNameGenerator, string $resourceClass, string $operationName = null)
     {
         if (null !== $operationName) {
-            $context = ['collection_operation_name' => $operationName];
+            $options = ['collection_operation_name' => $operationName];
```

Comment:
What do you think of `$propertyMetadataOptions`? (same everywhere, except where it's obvious)